src/services/arweave/SimpleArweaveDataProvider.ts

```diff
@@ -108,9 +111,10 @@ export class SimpleArweaveDataProvider implements ArweaveDataProvider {
     requiredTags?: { [x: string]: string[] };
   }): Promise<void> {
     // validate tx exists, their may be better ways to do this
+    // todo: implement http code error proccesor/handler
     const { status } = await this.getTransactionHeaders(id);
-    if (status !== 200) {
-      throw Error('Contract ID not found. Try again.');
+    if (!status.ok) {
+      throw Error(`Contract ID not found. Try again. Status: ${status}`);
```

Comment:
this is showing errors even on 200's....
<img width="862" alt="image" src="https://user-images.githubusercontent.com/12800001/217668355-55c59fd2-e22f-4686-92eb-d798bc0a6ae4.png">